packages/suite/src/components/suite/images/DeviceConfirmImage.tsx

```diff
@@ -4,12 +4,11 @@ import { Image, ImageProps } from '@trezor/components';
 import { ImageType } from '@trezor/components/src/components/Image/Image';
 import { getDeviceModel } from '@trezor/device-utils';
 
-const getImage = (majorVersion: '1' | 'T'): ImageType => {
+const getImage = (majorVersion: string): ImageType => {
```

Comment:
please rename `majorVersion` in the followup you are going to make for model R support in suite :D